pkg/reconciler/openshift/tektonaddon/tektonaddon.go

```diff
@@ -185,7 +185,9 @@ func (r *Reconciler) ReconcileKind(ctx context.Context, ta *v1alpha1.TektonAddon
 		}
 	} else {
 		// if disabled then delete the installer Set if exist
-		return r.deleteInstallerSet(ctx, ta, clusterTaskInstallerSet)
+		if err := r.deleteInstallerSet(ctx, ta, clusterTaskInstallerSet); err != nil {
+			return err
+		}
```

Comment:
nit: is this change really necessary.

Reply:
yeah it is required, if clustertasks are disabled then rest of the code was not executed 
so we only return if there is some err while deleting

Reply:
oh.. i see. it is about `not` returning nil. and not about returning error. 👍